spec/contact_spec.rb

```diff
@@ -50,5 +50,17 @@
         end
       end
     end
+
+    describe "#update_without_vid" do
+      let(:contact_data) do
+        Hubspotter::Contact.update_without_vid(
+          ContactFoundry.duplicate_email_properties.merge({resume: "resume link!"})
+        )
+      end
+
+      it "updates resume without knowing the vid" do
+        expect(contact_data).to include(:vid)
```

Comment:
A better test would be to load the user, update the resume (`expected = resume + "foo"`) and ensure that when you reload the user the updated resume content is reflected.  All this can be recorded in a cassette.